source/config/configFlags.py

```diff
@@ -6,7 +6,90 @@
 """Flags used for the configuration."""
 
 from enum import unique
-from utils.displayString import DisplayStringIntEnum
+from utils.displayString import DisplayStringIntEnum, DisplayStringStrEnum
+
+
+@unique
+class ShowMessages(DisplayStringIntEnum):
+	"""Enumeration containing the possible config values for "Show messages" option in braille settings.
+	
+	Use ShowMessages.MEMBER.value to compare with the config;
+	use ShowMessages.MEMBER.displayString in the UI for a translatable description of this member.
+	"""
+	
+	DISABLED = 0
+	USE_TIMEOUT = 1
+	SHOW_INDEFINITELY = 2
+	
+	@property
+	def _displayStringLabels(self):
+		return {
+			# Translators: One of the show states of braille messages
+			# (the disabled mode turns off showing of braille messages completely).
+			ShowMessages.DISABLED: _("Disabled"),
+			# Translators: One of the show states of braille messages
+			# (the timeout mode shows messages for the specific time).
+			ShowMessages.USE_TIMEOUT: _("Use timeout"),
+			# Translators: One of the show states of braille messages
+			# (the indefinitely mode prevents braille messages from disappearing automatically).
+			ShowMessages.SHOW_INDEFINITELY: _("Show indefinitely"),
+		}
+
+
+@unique
+class TetherTo(DisplayStringStrEnum):
+	"""Enumeration containing the possible config values for "Tether to" option in braille settings.
+	
+	Use TetherTo.MEMBER.value to compare with the config;
+	use TetherTo.MEMBER.displayString in the UI for a translatable description of this member.
+	"""
+	
+	AUTO = "auto"
+	FOCUS = "focus"
+	REVIEW = "review"
+	
+	@property
+	def _displayStringLabels(self):
+		return {
+			# Translators: The label for a braille setting indicating that braille should be
+			# tethered to focus or review cursor automatically.
+			TetherTo.AUTO: _("automatically"),
+			# Translators: The label for a braille setting indicating that braille should be tethered to focus.
+			TetherTo.FOCUS: _("to focus"),
+			# Translators: The label for a braille setting indicating that braille should be tethered
+			# to the review cursor.
+			TetherTo.REVIEW: _("to review"),
+		}
+
+
+@unique
+class ReportLineIndentation(DisplayStringIntEnum):
+	"""Enumeration containing the possible config values to report line indent.
+	
+	Use ReportLineIndentation.MEMBER.value to compare with the config;
+	use ReportLineIndentation.MEMBER.displayString in the UI for a translatable description of this member.
+	"""
+	
+	OFF = 0
+	SPEECH = 1
+	TONES = 2
+	SPEECH_AND_TONES = 3
+	
+	@property
+	def _displayStringLabels(self):
+		return {
+			# Translators: A choice in a combo box in the document formatting dialog to report No line Indentation.
+			ReportLineIndentation.OFF: _("Off"),
+			# Translators: A choice in a combo box in the document formatting dialog to report indentation
+			# with Speech.
+			ReportLineIndentation.SPEECH: pgettext('line indentation setting', "Speech"),
```

Comment:
Yes. I have taken the strings as originally defined in `settingsDialogs.py`, so that nothing changes in the translation files.

Probably `pgettext` has been used for `speech` because the translation of this word can vary depending on the context.

What is the goal of your remark? We may use `pgettext` for the 4 options if you wish.